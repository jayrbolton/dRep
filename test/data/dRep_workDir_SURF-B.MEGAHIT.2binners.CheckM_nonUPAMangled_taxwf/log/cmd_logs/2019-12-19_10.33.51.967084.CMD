prodigal -i /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.044.fasta -d /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.044.fasta.fna -a /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.044.fasta.faa -m -p meta
