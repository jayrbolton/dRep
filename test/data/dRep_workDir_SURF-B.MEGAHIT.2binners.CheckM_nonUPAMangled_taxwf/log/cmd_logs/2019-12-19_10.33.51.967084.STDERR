-------------------------------------
PRODIGAL v2.6.3 [February, 2016]         
Univ of Tenn / Oak Ridge National Lab
Doug Hyatt, Loren Hauser, et al.     
-------------------------------------
Request:  Metagenomic, Phase:  Training
Initializing training files...done!
-------------------------------------
Request:  Metagenomic, Phase:  Gene Finding
Finding genes in sequence #1 (116187 bp)...done!
Finding genes in sequence #2 (159613 bp)...done!
Finding genes in sequence #3 (131247 bp)...done!
Finding genes in sequence #4 (127427 bp)...done!
Finding genes in sequence #5 (411266 bp)...done!
Finding genes in sequence #6 (73174 bp)...done!
Finding genes in sequence #7 (1222178 bp)...done!
Finding genes in sequence #8 (19846 bp)...done!
Finding genes in sequence #9 (22601 bp)...done!
